modules/campaignion_supporter/src/Supporter.php

```diff
@@ -136,6 +136,8 @@ public function exporter($type, $language) {
         $map['SOURCE'] = new TagField('source_tag');
         $map['CAMPAIGN'] = new TagField('campaign_tag');
         $map['TAGS'] = new TagsField('supporter_tags', TRUE);
+        $map['ONETIMEVAL'] = new WrapperField('field_onetimeval');
+        $map['REGVAL'] = new WrapperField('field_regval');
```

Comment:
The fields don’t seem to exist in the starterkit. THere is no point in adding exporters for them if they don’t exist.